src/main/resources/subscriptionsTemplate.json

```diff
@@ -0,0 +1,58 @@
+[{
```

Comment:
This file is already in the project file structure: src/test/resources/subscription.json.
So now we have duplicates, since this has been merged. Updates so we have only one subscription template file.

Reply:
I will remove it in my coming PR today!